packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -198,9 +226,45 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                 }
             },
             // eth addresses are valid without checksum but Trezor displays them as checksummed
-            checksum: (value: string) => {
-                if (networkType === 'ethereum' && !checkAddressCheckSum(value)) {
-                    return translationString('RECIPIENT_IS_NOT_VALID');
+            checksum: async (address: string) => {
+                if (networkType === 'ethereum') {
+                    const updatedValues = addValue(address); // Get updated values immediately
+                    const params = {
+                        descriptor: address,
+                        coin: symbol,
+                    };
+
+                    //they are not the same but the second one is checksum of the first one, in this case show the info that the address was automatically checksummed
+                    if (
+                        updatedValues &&
+                        updatedValues[0] &&
+                        updatedValues[1] &&
+                        updatedValues[0] !== updatedValues[1] &&
+                        toChecksumAddress(updatedValues[0]) === updatedValues[1]
+                    ) {
+                        setAddressChecksumInfo({
+                            text: 'Address converted to checksum format.',
```

Comment:
This string is not used at all, why not calling this `addressChecksumInfo` -> `hasAddressBeenChecksummed` boolean?

Reply:
Yep, just did this myself too :D